themes/default/content/migrate/terraform.md

```diff
@@ -49,4 +49,134 @@ contact_us_form:
             boosting productivity and taking our business to any cloud environment that our customers need. We
```

Comment:
what is all of this?

Reply:
Ah, I'll remove this in the follow-up -- it was data that was used to populate the template. I just overlooked it. (It's no longer used.)